app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/Concert.java

```diff
@@ -0,0 +1,79 @@
+package ch.epfl.sweng.eventmanager.repository.data;
+
+
+import java.util.Date;
+
+/**
+ * Class describing a single concert in an event. The class is for the moment only used by the
+ * ScheduleActivity.
+ */
+public final class Concert {
```

Comment:
This class should probably not be named this way, as we don't only have concerts in our events. Should be something more generic, like `ScheduledEvent` or `ScheduledItem`